### ISSUE TITLE
pg port error

### ISSUE THREAD

**Issue by ydrea** [opencontext/arches-via-docker#6]
hi!

```
Error response from daemon: driver failed programming external connectivity on endpoint arches (209353b2d6c02b9a98f73add0682595c9e45ae86b548b28a00f4e92f10c612a9): failed to bind port 0.0.0.0:5432/tcp: Error starting userland proxy: listen tcp4 0.0.0.0:5432: bind: address already in use
```

error caused by docker compose up --build with edited pg access data:

docker-compose.yml
"    environment:
      - PGUSER=suhozid
      - POSTGRES_USER=suhozid
      - POSTGRES_PASSWORD=******************
 "
.env
"
PGUSERNAME=suhozid
PGPASSWORD=***********************
PGDBNAME=suhozid
PGHOST=arches_db
PGPORT=5432
ESHOST=arches_elasticsearch
ESPORT=9200
DJANGO_MODE=DEV
DJANGO_DEBUG=False
"

in other words - where does docker receive PGDBNAME?

**Comment by ydrea** [comment]
resolved
